src/header/header-buttons.scss

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../side-navigation/index';
+
+@mixin iui-header-brand {
+  height: 100%;
+  display: inline-flex;
+  align-items: center;
+  flex-shrink: 0;
+  white-space: nowrap;
+  background-color: transparent;
+  box-sizing: border-box;
+
+  @at-root {
+    a#{&}, // rendered as anchor element
+    button#{&}, // rendered as button element
+    &[role='button'] {
+      @include iui-focus;
+      cursor: pointer;
+      -webkit-tap-highlight-color: transparent;
+      transition: background-color $iui-speed-fast ease-out;
+      @media (forced-colors: active) {
+        border: 1px solid transparent;
+        border-bottom-width: 0;
+      }
+
+      // #region resets for button
+      @include iui-reset;
+      font-family: inherit;
+      background-color: unset;
+      color: unset;
+      // #endregion
+
+      &:hover {
+        background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+      }
+    }
+  }
+}
+
+@mixin iui-header-brand-icon {
+  width: $iui-icons-large;
+  height: $iui-icons-large;
+  margin: 0 $iui-side-navigation-icon-margins;
+  fill: var(--iui-icons-color-actionable);
```

Comment:
This might be an exception to the rule because it is a brand icon and changing it to the non-actionable fill looks wrong to me.
![Screen Shot 2022-05-19 at 11 50 35 AM](https://user-images.githubusercontent.com/849817/169342820-b55ba576-37e7-4506-95da-b4b4adf98dae.png)